bin/load_db_scxa_analytics.sh

```diff
@@ -54,15 +52,17 @@ matrixMarket2csv.js -m $matrix_path \
                     -s 50000 \
                     -o $EXPERIMENT_MATRICES_PATH/expression2load.csv
 
-# Load data into partition table
-sed "s/<EXP-ACCESSION>/$lc_exp_acc/" $postgres_scripts_dir/03-load_data.sql.template | \
+
+# Create and load partition table in the same transaction.
+sed "s/<EXP-ACCESSION>/$lc_exp_acc/" $postgres_scripts_dir/02-create_and_load_partition_table.sql.template | \
     sed "s+<PATH-TO-DATA>+$EXPERIMENT_MATRICES_PATH/expression2load.csv+" | \
     psql -v ON_ERROR_STOP=1 $dbConnection
 
 rm $EXPERIMENT_MATRICES_PATH/expression2load.csv
 
 # Create primary key.
-sed "s/<EXP-ACCESSION>/$lc_exp_acc/g" $postgres_scripts_dir/04-build_pk.sql.template | \
+echo "Create PK"
+time sed "s/<EXP-ACCESSION>/$lc_exp_acc/g" $postgres_scripts_dir/04-build_pk.sql.template | \
```

Comment:
Are you intending this time to remain, or is it a debugging relic?

Reply:
This I want to keep to get the values on the current loading rounds, to see whether there is a substantial time in creating the PK that would merit the sorts that we were discussing yesterday.

Reply:
Gotcha